### ISSUE TITLE
'See more' links in Add-on Category pages are not consistent with the links in the homepage

### ISSUE THREAD

**Issue by AlexandraMoga** [mozilla/addons-frontend#3854]
Steps to reproduce:

1. Open an Extension/Theme Category page (i.e. https://addons-dev.allizom.org/en-US/firefox/extensions/alerts-updates/)
2. Observe the 'See more' links in the Featured/Top rated... shelves

Expected results:
The links have the same names as in the homepage (i.e. 'Featured extensions' shelf should have a 'See more featured extensions' link)

Actual result
The Category shelves have only a 'See more' link

Note: reproduced on all AMO servers, with FF56 (Win10 and Android 7.0)
Ref [#3633](https://github.com/mozilla/addons-frontend/issues/3633#issuecomment-342882829)
![image](https://user-images.githubusercontent.com/31961530/32598982-cd442162-c543-11e7-9fa7-079a5726e801.png)

**Comment by muffinresearch** [comment]
This is also an accessibility concern.

**Comment by AlexandraMoga** [comment]
I could see the changes applied in Extension category pages, but not in Themes..

![image](https://user-images.githubusercontent.com/31961530/32604949-36c10fce-c558-11e7-91cf-c07c0a96f1f8.png)

**Comment by willdurand** [comment]
Thanks @AlexandraMoga. I've created #3858 for that.

**Comment by willdurand** [comment]
@tsl143 would you like to work on #3858?

**Comment by AlexandraMoga** [comment]
Verified as fixed on AMO-dev (Win10 and Android 7.0)

![image](https://user-images.githubusercontent.com/31961530/32606329-9985a94e-c55d-11e7-9907-45d985f6e407.png)
